lib/src/components/NxPolicyViolationIndicator/NxPolicyViolationIndicator.scss

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-policy-violation-indicator {
+  align-items: center;
+  background-color: #fff;
+  border: 1px solid;
+  border-radius: $nx-border-radius;
+  display: flex;
+  height: 30px;
+  margin: 0 0 $nx-spacing-xs $nx-spacing-md;
+  max-width: 860px;
+  width: fit-content;
+}
+
+.nx-policy-violation-indicator__badge {
+  border-top-left-radius: $nx-border-radius;
+  border-bottom-left-radius: $nx-border-radius;
+  height: 30px;
+
+  .nx-icon {
+    color: #fff;
+    vertical-align: -5px;
+  }
+}
+
+.nx-policy-violation-indicator__text {
+  @include semi-bold();
+
+  border-top-right-radius: $nx-border-radius;
+  border-bottom-right-radius: $nx-border-radius;
+  display: inline-block;
+  font-size: $nx-font-size-s;
+  margin: 0 $nx-spacing-md;
+  text-transform: uppercase;
+  vertical-align: middle;
+}
+
+.nx-policy-violation-indicator--unspecified {
+  border-color: $nx-threat-color-unspecified;
+
+  .nx-policy-violation-indicator__badge {
+    background-color: $nx-threat-color-unspecified;
```

Comment:
If you wanted to, these could be a good time to play with CSS custom properties/variables. The border color and background color would be defined just once in the non-modifier CSS blocks (using a custom property), and then these modifier blocks would just set the custom property.

Reply:
> If you wanted to, these could be a good time to play with CSS custom properties/variables. The border color and background color would be defined just once in the non-modifier CSS blocks (using a custom property), and then these modifier blocks would just set the custom property.

Tricky but fun, thanks for the tip that got this across the finish line: https://github.com/sonatype/sonatype-react-shared-components/pull/229/commits/22cfc350d2aa4b3010f9bf38ead213a04e9dab04